src/app/shared/api-locator.service.ts

```diff
@@ -73,6 +73,7 @@ export class BaseApiLocatorService {
     if (this.envVars.get(key)) {
       return this.envVars.get(key);
     }
+
```

Comment:
This file change can be ignored as it has only a space difference and nothing significant.